content/en/change_tracking/_index.md

```diff
@@ -0,0 +1,117 @@
+- **Services**: No prerequisites.
+
+## Using Change Tracking
+
+Change Tracking is available on several pages in Datadog:
+
+### Monitor status page
+
+View and analyze changes from the [monitor status page][1].
+
+{{< img src="/change_tracking/change-tracking-monitor-status-page.png" alt="Change Tracking displayed on the Monitor Status Page" style="width:100%;" >}}
+
+
+To analyze changes from the monitor status page:
+
+1. Go to the monitor status page for the monitor you are analyzing.
+1. Locate the **Recent Changes** section.
+1. Use the **Recent Changes** timeline together with the **Status & History** graphs to correlate change events with the alert.
+1. Click the change indicator in the **Recent Changes** timeline to view more details about the change in the side panel.
+1. From the side panel, you can investigate more details and take the following actions:
+   - View the deployment in your CI/CD system.
+   - View the latest commits in your repository.
+   - Compare changes between deployments to identify potential issues.
+   - Set context-aware, **Custom Links** for quick access to relevant resources.
+
+### Services
+
+View and analyze changes from the [service page][2].
+
+{{< img src="/change_tracking/change-tracking-service-page.png" alt="Recent Changes component within the Service Summary section with dependency changes shown" style="width:100%;" >}}
+
+To analyze changes from the service page:
+
+1. Navigate to the service page you want to investigate.
+1. Locate the changes timeline in the **Service Summary** section.
+1. Use the dropdown to view either:
+   - Changes limited to the specific service (**Changes by Service**)
+   - Changes to the specific service and dependent services that might impact this service (**Changes by Service & Dependencies**)
+1. Click the change indicator to view detailed information and take remediation actions.
+
+### Dashboards
+
+View and analyze changes from any [dashboard][3].
+
+{{< img src="/change_tracking/change-tracking-dashboard-show-overlays-active.png" alt="Change Tracking displayed on the Dashboard" style="width:100%;" >}}
+
+To analyze changes from dashboards:
+
+1. Navigate to your dashboard.
+2. Click **Show Overlays** at the top of the page to enable the change timeline and change overlays on supported widgets.
+3. Hover over any change indicator or overlay to view a summary of the change.
+4. Click the change indicator or overlay to view detailed information and take remediation actions.
+
+{{< img src="/change_tracking/change-tracking-deployment-details-side-panel.png" alt="Change Tracking side panel open" style="width:100%;" >}}
+
+## Tracked changes
+
+Change Tracking follows these types of changes across your infrastructure:
+
+| Change Type | Tracking Requirements |
+| ----------- | ----------- |
+| Code Deployments (APM) | Works out of the box for APM customers as the agent injects the version if not provided.
```

Comment:
"the agent injects the version if not provided."
To my knowledge that is not true. It is only the case in container environement. We should link to the unified service tagging documentation and just add as a requiment that a version is available on services